userguide/tutorials/metrics-newrelic.adoc

```diff
@@ -0,0 +1,172 @@
+= Managing Metrics with New Relic
+
+:imagesdir: https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws
+
+
+== Single Tier and Multitier Implementations
+
+Measurements and metrics are an important part of managing your Kill
+Bill system to be sure that performance is optimized and any problems
+can be detected early. Kill Bill collects over 1,300 metrics to show you
+every conceivable view of your system's behavior.
```

Comment:
I'm still curious how users access a list of what these metrics are. @pierre showed me where they were, but said we shouldn't publicly announce it. Do users (clients) just ask us privately, then? Or do they dig around themselves until they find it?